src/readsbml.jl

```diff
@@ -228,8 +261,16 @@ function extractModel(mdl::VPtr)::SBML.Model
             )
         end
 
+        ic = nothing
+        if ccall(sbml(:Species_isSetInitialConcentration), Cint, (VPtr,), sp) != 0
+            ic = (
+                ccall(sbml(:Species_getInitialConcentration), Cdouble, (VPtr,), sp),
+                get_string(sp, :Species_getSubstanceUnits),
+            )
+        end
+
         species[get_string(sp, :Species_getId)] = Species(
-            get_optional_string(sp, :Species_getName),
+            get_optional_string(sp, :Species_getName),  # PL: wondering if we should use Id instead of name here
```

Comment:
identifiers are parsed as identifiers (ie keys), that cannot be confused with (completely optional and computtionally irrelevant) "pretty name".